product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -96,7 +96,12 @@ The following options aren't available when creating your cluster:
        !!!tip
        To maximize your disk size for AWS, select R5b as your instance and then io2 Block Express as your storage to get a maximum disk size of 64 TB and 256,000 IOPS.
   
-1.  In the **Storage** section, from the **Volume Type** list, select your volume type. 
+1.  In the **Storage** section:
```

Comment:
We should also let customer know that once they add WAL volume (existing or new PG) they cannot remove it later from the cluster settings. 

cc: @mbiagetti and @neel5481